reference-architectures/deploy-arch-ibm-slz-ocp.md

```diff
@@ -0,0 +1,124 @@
+https://test.cloud.ibm.com/docs/solution-as-code?topic=solution-as-code-naming-guidance.
+-->
+
+# OpenShift Container Platform on VPC
+{: #ocp-ra}
+{: toc-content-type="reference-architecture"}
+{: toc-industry="Banking,FinancialSector"}
+{: toc-use-case="Cybersecurity"}
+{: toc-compliance="FedRAMP"}
+
+OpenShift Container Platform on VPC for regulated industries is a deployable architecture solution that is based on the IBM Cloud for Financial Services reference architecture. It creates secure and compliant Red Hat OpenShift Container Platform workload clusters on a Virtual Private Cloud (VPC) network.
+
+## Architecture diagram
+{: #ra-ocp-architecture-diagram}
+
+![Architecture diagram of the OpenShift Container Platform on VPC deployable architecture.](roks.drawio.svg "Architecture diagram of OpenShift Container Platform on VPC for regulated industries deployable architecture"){: caption="Figure 1. Single region architecture diagram for Red Hat OpenShift Container Platform on VPC on IBM Cloud" caption-side="bottom"}
+
+## Design requirements
+{: #ra-ocp-design-requirements}
+
+![Design requirements for Secure infrastructure on VPC for regulated industries](heat-map-deploy-arch-slz-ocp.svg "Design requirements"){: caption="Figure 2. Scope of the design requirements" caption-side="bottom"}
+
+<!--
+TODO: Add the typical use case for the architecture.
+The use case might include the motivation for the architecture composition,
+business challenge, or target cloud environments.
+-->
+
+## Components
+{: #ra-ocp-components}
+
+### VPC architecture decisions
+{: #ra-ocp-components-arch}
+
+| Requirement | Component | Reasons for choice | Alternative choice |
+|-------------|-----------|--------------------|--------------------|
+| * Ensure public internet connectivity  \n * Isolate most OpenShift clusters so that they are not reachable directly from the public internet | Edge VPC service|Create a separate VPC service where public internet connectivity is allowed to be configured | |
+| Provide compute, storage, and network services to enable the application application provider's administrators to monitor, operate, and maintain the environment | Management VPC service | Create a separate VPC service where SSH connectivity from outside is allowed | |
+| Provide compute, storage, and network services to support hosted applications and operations that deliver services to the consumer | Workload VPC service | Create a separate VPC service as an isolated environment, without direct public internet connectivity and without direct SSH access | |
+| Create a virtual server instance that can act as an internet proxy server | Proxy server VPC instance | Create a Linux VPC instance that can act as a proxy server. Configure ACL and security group rules to allow public internet traffic over proxy that uses default proxy ports (3828) | Configure application load balancer to act as proxy server manually |
```

Comment:
no virtual instance in this architecture.  Just OpenShift clusters

Reply:
removed row